### PR TITLE
[Imaging QC + Behavioural QC] Cleanup integration test code 

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes

Use changes from recent PRs #5619, #6041, and #6046 to simplify the integration tests for these modules.

### CODE REVIEW

**Comment by johnsaigle** [comment]
Blocked by #6041

File: SQL/0000-00-01-Permission.sql [hunk 1]
```diff
@@ -75,7 +75,6 @@ INSERT INTO `permissions` VALUES
     (25,'dicom_archive_view_allsites', 'Across all sites view Dicom Archive module and pages', '2'),
     (28,'instrument_builder', 'Instrument Builder: Create and Edit instrument forms', '2'),
     (29,'data_dict_edit','Edit Data Dictionary','2'),
-    (30,'quality_control','Quality Control access','2'),
```
Comment (kongtiaowang) [inline SQL/0000-00-01-Permission.sql:78]: Why did you remove this permission?

Comment (kongtiaowang) [inline SQL/0000-00-01-Permission.sql:78]: Since 30 has been remove. the order sould be changed too. (not 29, 31, 32)

File: raisinbread/RB_files/RB_permissions.sql [hunk 2]
```diff
@@ -63,5 +63,8 @@ INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES
 INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (61,'electrophysiology_browser_view_site','View own site Electrophysiology Browser pages',2);
 INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (62,'module_manager_view','Module Manager: access the module',2);
 INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (63,'module_manager_edit','Module Manager: edit installed modules',2);
+INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (64,'survey_accounts_view','Survey Accounts: view module',2);
```
Comment (kongtiaowang) [inline raisinbread/RB_files/RB_permissions.sql:66]: line 30 should be removed.
 (30,'quality_control','Quality Control access',2);

File: SQL/New_patches/2020-02-10_NewModulePermissions.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO permissions (code,description,categoryID) VALUES
```
Comment (kongtiaowang) [inline SQL/New_patches/2020-02-10_NewModulePermissions.sql:1]: Missing remove old permission (30,'quality_control','Quality Control access','2')

**Comment by johnsaigle** [comment]
@kongtiaowang Thanks for taking a look. I'll rebase this PR after the one blocking it gets merged. That should take care of the errors.

**Comment by ridz1208** [comment]
@AlexandraLivadas please issue this one from your fork and fix conflicts  and incorporate changes.

**Comment by AlexandraLivadas** [comment]
#6041 was merged so I have removed the Blocked label

**Comment by cmadjar** [comment]
@kongtiaowang can you re-review?

**Comment by driusan** [comment]
@kongtiaowang can you rereview? I think all your changes were done.

File: raisinbread/RB_files/RB_permissions.sql [hunk 4]
```diff
@@ -29,45 +29,44 @@ INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES
 INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (27,'reliability_swap_candidates','Swap Reliability candidates across all sites',2);
 INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (28,'instrument_builder','Instrument Builder: Create and Edit instrument forms',2);
 INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (29,'data_dict_edit','Edit Data Dictionary',2);
-INSERT INTO `permissions` (`permID`, `code`, `description`, `categoryID`) VALUES (30,'quality_control','Quality Control access',2);
```
Comment (driusan) [inline raisinbread/RB_files/RB_permissions.sql:32]: This arbitrary renumbering of permissions is going to break all the foreign keys to the permissions table in raisinbread.

Comment (driusan) [reply raisinbread/RB_files/RB_permissions.sql:32]: @AlexandraLivadas can you remove the changes to the permission numbers in RB and the schema? They're unnecessary.
